test/erlcloud_aws_tests.erl

```diff
@@ -29,7 +29,7 @@ config() ->
                 retry_num = 3}.
 
 request_default_test(_) ->
-    ok = erlcloud_aws:aws_request(get, "host", "/", [], "id", "key"),
+    _Body = erlcloud_aws:aws_request(get, "host", "/", [], "id", "key"),
```

Comment:
I don't understand how these could have been passing, since I don't see `:aws_request` ever returning `ok`.

Reply:
that is... very weird.

Reply:
I wonder if these are working right at all? ending your test function with _test triggers special treatment with eunit, and I kind of wonder if that is short-circuiting something in eunit. maybe it would be better to make these functions in the `request_test_` group _prefixed_ with `test_` rather than _suffixed_ with `_test`.

Reply:
Done in 7f77b78.

I tried to revert to `ok`, prior to the change, and tests pass.
With `ok` but performing the function name changes the tests pass.

I'll look further into it, since we are getting ok as a result, for sure.

Reply:
It's coming from line 19:
```
meck:expect(erlcloud_httpc, request, fun(_,_,_,_,_,_) -> {ok, {{200, "OK"}, [], ok}} end)
                                                                                ^ this!
```

Reply:
Hah! wow. Ever feel like.... you're being mocked? 😉

Reply:
I made it more visible (easier to find) in c187237. I used example <<"OkBody">> since it was already present elsewhere.

Reply:
> Hah! wow. Ever feel like.... you're being mocked? 😉

Many times :D